chanjo/cli/base.py

```diff
@@ -6,7 +6,10 @@
 """
 import logging
 import os
-import pkg_resources
+try:
+    from importlib.metadata import entry_points
+except ImportError:  # Backport support for importlib metadata on Python 3.7
```

Comment:
Now I understand, it's for the Dockerfile!

Reply:
It doesn't hurt much, other than perhaps that we install the compat libs without using them, so I guess we just keep it.